src/TurbulenceConvection/update_aux.jl

```diff
@@ -141,7 +140,8 @@ function update_aux!(
     end
 
     @. aux_en.e_kin =
-        LA.norm_sqr(C123(prog_gm_uₕ) + C123(Ic(wvec(aux_en_f.w)))) / 2
+        LinearAlgebra.norm_sqr(prog_gm_uₕ) +
+        ᶜinterp(LinearAlgebra.norm_sqr(aux_en_f.w)) / 2
```

Comment:
This KE calculation seems to be repeated many times. It would be good to isolate it into a function.